src/components/Sidebar/Sidebar.jsx

```diff
@@ -35,7 +37,13 @@ class Sidebar extends React.Component {
         }
     }
 
+    handleLogoutClick() {
+        localStorage.clear();
```

Comment:
normally we shouldn't clear all of the client browser's local-storage. Just get rid of our token `billsnap-token`